### ISSUE TITLE
Another jedi error that leads to kernel fatal

### ISSUE THREAD

**Issue by leopsidom** [pyodide/pyodide#1793]
Not sure if we currently have a solution for this. But here is another jedi case that leads to pyodide fatal error:

```
import pandas as pd
import jedi
s = pd.Series([1,2,3])
script = jedi.Interpreter("s.index.", [globals()])
script.complete()
```

> Pyodide has suffered a fatal error. Please report this to the Pyodide maintainers.
> The cause of the fatal error was:
> InternalError: too much recursion

Tested in REPL: https://pyodide.org/en/stable/console.html in chrome (92.0.4515.131) and firefox (78.10.1esr).

**Comment by hoodmane** [comment]
I cannot reproduce in Firefox 91, but can reproduce in Chrome 92. Has a lot to do with the prevalence of function pointer calls on the stack. I patched CPython to remove function pointer calls from normal Python to Python function calls, but for some reason these stack frames have more of the fat fpcast calls. We probably won't be able to fix this until #1677 is merged.

**Comment by hoodmane** [comment]
Closed by #2019.
